keyboards/ktec/ergodone/expander.c

```diff
@@ -20,7 +22,7 @@ void expander_init(void)
 void expander_scan(void)
 {
   dprintf("expander status: %d ... ", expander_status);
-  uint8_t ret = i2c_start(EXPANDER_ADDR | I2C_WRITE);
+  uint8_t ret = i2c_start(EXPANDER_ADDR | I2C_WRITE, I2C_TIMEOUT);
```

Comment:
This code remains effectively AVR-only, because on ChibiOS `i2c_start()` does not actually communicate with the I2C slave, which is what the code apparently tries to do. I'm not sure whether ChibiOS supports zero-length I2C transfers, so if you really want to get rid of low level I2C API here, you may need to replace this with an attempt to read some expander register.

Reply:
Will be completed in a future PR.